websites.yml

```diff
@@ -9,9 +9,22 @@ urls:
   medium: https://{w}.com/@{u}
   openhub: https://www.{w}.net/accounts/{u}
   opensuse: https://connect.{w}.org/pg/profile/{u}
-  pinterest: https://in.{w}.com/{u}
+  pinterest: https://www.{w}.com/{u}
```

Comment:
I think it should be put in seperate commit.

Reply:
strictly speaking , it should be a separate commit, but we would only request that of students after they have done a few tasks and are starting to become a regular contributor.

Reply:
Done.

Reply:
@jayvdb